haiku-apps/genio/genio-4.0.recipe

```diff
@@ -11,15 +11,15 @@ declaration, quick fix, format, rename)
 * User templates for quickly creating new files and projects
 * Rich editor with many features, like syntax highlighting
 * Full screen and Focus mode"
-HOMEPAGE="https://github.com/Genio-The-Haiku-IDE/Genio/releases"
+HOMEPAGE="https://codeberg.org/Genio/Genio/releases"
 COPYRIGHT="2022-2025 The Genio Team"
 LICENSE="BSD (3-clause)
 	MIT"
-REVISION="1"
-SOURCE_URI="https://github.com/Genio-The-Haiku-IDE/Genio/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="33de5cb06b625aa3c0281b5696801a54910147d2848beda35e17624504faa88e"
+REVISION="2"
```

Comment:
IMO, as this change doesn't offers any change to end users, REVISION should be kept at 1.

Otherwise, this could perhaps wait till next version update.